src/patternfly/components/ModalBox/docs/code.md

```diff
@@ -10,13 +10,16 @@ A modal box is a generic rectangular container that can be used to build modals.
 | `aria-labeledby="TitleID"` | `.pf-c-modal-box` | Gives the modal an accessible name by referring to the element that provides the dialog title. **Required** |
 | `aria-describedby="ContentID"` | `.pf-c-modal-box` | Gives the modal an accessible description by referring to the modal content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the modal. |
 | `aria-modal="true"` | `.pf-c-modal-box` | Tells assistive technologies that the windows underneath the current modal are not available for interaction. **Required**|
+| `aria-label="Close Dialog"` | `.pf-c-modal-box__close` | Provides an accessible name for the close button as it uses an icon instead of text. **Required**|
 
 ## Usage
 
 | Class                | Applied     | Outcome                                                                                                                                                                              |
 | -------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `.pf-c-modal-box`          | `<div>`     | Creates a box containing content                                                                                                                                                     |
-| `.pf-c-modal-box__header`          | `<div>`     | Creates the header of a box.                                                                                                                                                     |
-| `.pf-c-modal-box__body`          | `<div>`     | Creates the body of a box.  ** Required.**                                                                                                                                                   |
-| `.pf-c-modal-box__footer`          | `<div>`     | Creates the footer of a box.                                                                                                                                                     |
-| `.pf-m-lg`          | `.pf-c-modal-box`     | Creates a wide version of the modal.                                                                                                                                                     |
+| `.pf-c-modal-box` |               `<div>` |           Initiates a modal box. **Required** |
+| `.pf-c-modal-box__header` |       `<div>` |           Initiates a modal box header. A modal box header is **required** if there is no modal box body. |
+| `.pf-c-modal-box__header-title` | `<div>` |           Initiates a modal box header title. |
```

Comment:
Talking to @jgiardino about this further, if no header is present the 'aria-label' should be provided. Do we need to show this as an example? This would also show what the close button could look like outside the header.  There is an issue for this <a href="https://github.com/patternfly/patternfly-next/issues/445">here</a>